cli/flox/src/commands/auth.rs

```diff
@@ -88,15 +88,21 @@ pub async fn authorize(client: BasicClient, floxhub_url: &Url) -> Result<Credent
 
     let done = Arc::new(AtomicBool::default());
 
+    let verification_uri = details
+        .verification_uri_complete()
+        .expect("Verification URI is always provided by the auth server")
```

Comment:
I've seen two conventions for `expect`, both used in this codebase: a) explain what failed, b) explain why it can't fail. The Rust book suggests the latter. But maybe it should be a normative statement rather than descriptive (ie. "Verification URI must be provided by the auth server")